pkg/search/search.go

```diff
@@ -80,7 +82,17 @@ func (o *RepoOptions) Run(logger log.Logger, args []string) error {
 	}
 
 	search.SortScore(res)
-	data, err := o.applyConstraint(res)
+	// TODO(itxaka): What happens if we pass more than one annotation? Should we check for all?
+	// Mainly, is this an AND or an OR? Should we just iterate over all the annotations here and
+	// call applyAnnotationsConstraint with each so it works as an AND?
+	// First we need to check for annotations, otherwise the applyConstraint will only return the highest
+	// chart version for a given release
+	data, err := o.applyAnnotationsConstraint(res)
```

Comment:
since on line 95 we call `o.applConstraint(data)` with the data that we generate here, I suppose it's an AND?
I trust the tests :man_shrugging: .

Reply:
Goddamm, maybe this is not as explicit as I wanted to be. I know its an AND **currently**! But in the comment Im wondering about what we should do with this, should we treat as AND or maybe as OR? Should we only allow passing one search annotation? etc...

I will update the comment to make it clearer once the search PR is merged and I need to rebase this